src/commands/Pokemon/pokedex.ts

```diff
@@ -0,0 +1,176 @@
+
+				// In case there are multiple evolutionary paths
+				const otherFormeEvos = poke.evolutions.slice(1);
+				if (otherFormeEvos.length) {
+					evoChain = `${evoChain}, ${otherFormeEvos.map(oevo => `\`${oevo.species}\` (${oevo.evolutionLevel})`).join(', ')}`;
+				}
+
+				// If the direct evolution has another evolution (charmander -> charmeleon -> charizard)
+				if (poke.evolutions[0].evolutions?.length) {
+					evoChain = `${evoChain} → \`${toTitleCase(poke.evolutions[0].evolutions[0].species)}\` (${poke.evolutions[0].evolutions[0].evolutionLevel})`;
+				}
+			}
+
+			return message.sendEmbed(new MessageEmbed()
+				.setColor(this.resolveColour(poke.color))
+				.setAuthor(`#${poke.num} - ${toTitleCase(poke.species)}`, poke.sprite)
+				.setImage(poke.sprite)
+				.setThumbnail(POKEMON_EMBED_THUMBNAIL)
+				.addField(embedTranslations.types, poke.types.join(', '), true)
+				.addField(embedTranslations.abilities, abilities, true)
+				.addField(embedTranslations.genderRatio, genders, true)
+				.addField(embedTranslations.smogonTier, poke.smogonTier, true)
+				.addField(embedTranslations.height, `${poke.height}m`, true)
+				.addField(embedTranslations.weight, `${poke.weight}kg`, true)
+				.addField(embedTranslations.eggGroups, poke.eggGroups?.join(', ') || '', true)
+				.addBlankField(true)
+				.addField(embedTranslations.otherFormes, poke.otherFormes?.join(', ') || embedTranslations.none, true)
+				.addField(embedTranslations.evolutionaryLine, evoChain)
+				.addField(embedTranslations.baseStats, baseStats.join(', '))
+				.addField(embedTranslations.flavourText, `\`(${poke.flavorTexts[0].game})\` ${poke.flavorTexts[0].flavor}`)
+				.addField(embedTranslations.externalResources, [
+					`[Bulbapedia](${parseBulbapediaURL(poke.bulbapediaPage)} )`,
+					`[Serebii](${poke.serebiiPage})`,
+					`[Smogon](${poke.smogonPage})`
+				].join(' | ')));
+		} catch (err) {
+			throw message.language.tget('COMMAND_POKEDEX_QUERY_FAIL', pokemon);
+		}
+	}
+
+	private async fetchAPI(message: KlasaMessage, pokemon: string) {
+		try {
+			return fetch(POKEMON_GRAPHQL_API_URL, {
+				method: 'POST',
+				headers: {
+					'Content-Type': 'application/json'
+				},
+				body: JSON.stringify({
+					query: getPokemonDetailsByFuzzy(pokemon)
+				})
+			}, 'json') as Promise<GraphQLPokemonResponse<'getPokemonDetailsByFuzzy'>>;
+		} catch (err) {
+			throw message.language.tget('SYSTEM_QUERY_FAIL');
+		}
+	}
+
+	private resolveColour(col: string) {
+		switch (col) {
+			case 'Black':
+				return '#323232';
+			case 'Blue':
+				return '#257CFF';
+			case 'Brown':
+				return '#A3501A';
+			case 'Gray':
+				return '#969696';
+			case 'Green':
+				return '#3EFF4E';
+			case 'Pink':
+				return '#FF65A5';
+			case 'Purple':
+				return '#A63DE8';
+			case 'Red':
+				return '#FF3232';
+			case 'White':
+				return '#E1E1E1';
+			case 'Yellow':
+				return '#FFF359';
+			default:
+				return '#FF0000';
```

Comment:
Those should return hexadecimal numbers, which is what `MessageEmbed#setColor` actually expects. For instance, change `#323232` to `0x323232`, same thing for all others.